src/markdown-pages/docs/ecosystem/galasa-cli.md

```diff
@@ -0,0 +1,153 @@
+
+All the tests in the `test.yaml` portfolio from `dev.galasa.simbank.tests` will run on the z/OS LPAR `MV2C` in the `PLEX2` cluster. The tests from `dev.galasa.simbank.tests.two` will run on the z/OS LPAR `MV2D` in the `PLEX2` cluster when the `galasactl runs submit --portfolio test.yaml` command is run.
+
+```
+galasactl runs prepare 
+                        --portfolio test.yaml
+                        --stream BestSoFar
+                        --package dev.galasa.simbank.tests
+                        --override zos.default.lpar=MV2C
+                        --override zos.default.cluster=PLEX2
+galasactl runs prepare
+                        --portfolio test.yaml
+                        --append
+                        --stream BestSoFar
+                        --package dev.galasa.simbank.tests.two
+                        --override zos.default.lpar=MV2D
+                        --override zos.default.cluster=PLEX2
+```
+
+## Working with the `runs submit` command
+
+The following section provides a subset of examples of how you can use the `runs submit` command to complete various tasks, for example, getting help, submitting tests, and setting overrides.
+
+### Getting help
+
+Use the following commands to get more information about the command and command options.
+
+```
+galasactl --help
+galasactl runs --help
+galasactl runs runs --help
+```
+
+### Submitting tests from a portfolio
+
+The example command submits tests from the `test.yaml` portfolio, and specifies the following settings: 
+```
+galasactl runs submit
+                        --portfolio test.yaml
+                        --poll 5
+                        --progress 1
+                        --throttle 5
+```
+where: 
+- `poll` specifies the frequency in seconds that the CLI polls the Ecosystem for test run status. The default value is `30` seconds. 
+- `progress` specifies the frequency in minutes that the CLI reports the overall progress of the test runs. The default value is `5` minutes. A value of  `-1` or less disables progress reports.
+- `throttle` specifies the number of test runs that can be submitted in parallel. The default value is `3`. A value of `0` or less  prevents throttling.
+
+### Submitting tests without a portfolio
+
+You can use test class names to submit test runs without using a portfolio.
+
+The following command runs `SimBankIVT` and `BasicAccountCreditTest` from `dev.galasa.simbank.tests`. 
+
+```
+galasactl runs submit
+                        --class dev.galasa.simbank.tests/SimBankIVT
+                        --class dev.galasa.simbank.tests/BasicAccountCreditTest
+```
+
+### Setting overrides for all tests during a run
+
+The following command runs all the tests in the `test.yaml` portfolio are on the z/OS LPAR `MV2C` in the `PLEX2` cluster.
```

Comment:
Need to mention that the overrides in the portfolio take precedence over the overrised on the submit command.   This is so you can set general overrides on the submit,   but have specific class overrides in the portfolio

Reply:
updated